docs/Developers/Newcomers_Guide.rst

```diff
@@ -310,6 +310,7 @@ and send it. You have successfully edited your last commit!
 
 Where ``myfork`` is explained
 `here <http://api.coala.io/en/latest/Developers/Newcomers_Guide.html#step-4-creating-a-fork-and-testing-your-changes>`_.
+The **Pull Request** will automatically update with the newest changes. 
```

Comment:
why is pull request bold? I don't see this helping to understand this, however it does disturb typography.